float64 x
float64 y
float64 z
float64 occupancy
float64 time